Source/NonVisuals/DCSBIOSBindings/DCSBiosActionBindingCDU737.cs

```diff
@@ -0,0 +1,120 @@
+﻿//namespace NonVisuals.DCSBIOSBindings
```

Comment:
Is this Work in progress or can this file be removed ?

Reply:
removed